etc/inc/dyndns.class

```diff
@@ -1288,6 +1299,33 @@
 						$this->_debug($body);
 					}
 					break;
+				case 'dnsmadeeasy':
+					if (preg_match('/success/i', $data)) {
+						$status = "phpDynDNS({$this->_dnsHost}): (Success) IP Address Changed Successfully! (".$this->_dnsIP.")";
```

Comment:
In that case, all the other entries should use `gettext()` as well. I am reluctant to change them in this PR.